lsstseries/timeseries.py

```diff
@@ -162,7 +166,9 @@ def sf2(self, bins=None, band=None):
 
         Notes
         ----------
-        In case that no value for band is passed, the function is executed
+        In case that no value for band_to_calc is passed, the function is executed
         on all available bands.
         """
-        return calc_sf2(self.time, self.flux, self.flux_err, bins, self.band, band_to_calc=band)
+        lc_id = [self.meta['id']]*len(self.time)
```

Comment:
I tried testing code with a timeseries that I created from a dictionary (initial way we did it) and that fails because it has no id value. Can we make that it is possible to calculate sf2 for a single timeseries without id value? 

I see that in the test you manually insert id value - I am fine with that as long as it is hidden from the user.

Reply:
We should expect to have an ID populated through most of our timeseries ingest methods eventually, but agree that there's no downside to having it work when it's not specified here.